src/mapper/postTransformations.ts

```diff
@@ -0,0 +1,131 @@
+import cytoscape, { ElementDefinition, Position } from 'cytoscape';
+import { NodeType } from '../models/nodeType';
+import { getSymbol } from '../symbol-api/getSymbol';
+import { NodeSymbol } from '../symbol-api/types/NodeSymbol';
+import { SymbolRotation } from '../symbol-api/types/SymbolRotation';
```

Comment:
mb add `index.ts` to make it more beautiful?

Reply:
👍